core/spec/models/spree/taxon_image_spec.rb

```diff
@@ -6,6 +6,24 @@
     let(:image_file) { File.open(Spree::Core::Engine.root + 'spec/fixtures' + 'thinking-cat.jpg') }
     let(:text_file) { File.open(Spree::Core::Engine.root + 'spec/fixtures' + 'text-file.txt') }
 
+    it 'has attachment present' do
```

Comment:
Right now, these tests are dependant on `Rails.application.config.use_paperclip` setting and only one test branch will be covered. Maybe we could set the value and test multiple conditions?

Reply:
@piotrleniec-spark both active storage and paperclip are tested inside those tests. All CI builds are tested against AS and paperclip as a separate processes